RELEASE-NOTES.md

```diff
@@ -1,5 +1,50 @@
 # Prefect Release Notes
 
+## Release 2.7.11
+
+### Exciting New Features 🎉
+- Add refactor of results into a standalone artifact table — https://github.com/PrefectHQ/prefect/pull/8164
+- Add Prefect Cloud Events schema and clients. — https://github.com/PrefectHQ/prefect/pull/8357
```

Comment:
@chrisguidry how should we announce this? Does this make sense to place in experimental for now or do we want to make a splash or... ?

Reply:
Yes, at most, let's say "Experimental".  We don't have it hooked up to the main Client yet.  I'd say we can either tag it experimental or quietly drop/de-emphasize it.  @WillRaphaelson what do you think?

Reply:
Sorry thanks for catching this guidry, yes experimental please!